package/gluon-mesh-vpn-tunneldigger/files/usr/bin/tunneldigger-watchdog

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/lua
+
+function restartTunneldigger()
+	io.popen('logger -t tunneldigger-watchdog "Restarting Tunneldigger."')
+	os.execute("/etc/init.d/tunneldigger restart")
+end
+
+function readPidFile()
+	local pidFile = io.open("/var/run/tunneldigger.mesh-vpn.pid", "r")
+	local returnPid = "0"
+	if pidFile~=nil then 
+		returnPid = pidFile:read("*l") 
+		io.close(pidFile)
+	end
+	return returnPid
+end
+
+function getNeighbourCount()
+        local list = io.lines("/sys/kernel/debug/batman_adv/bat0/originators")
```

Comment:
the debugfs is deprecated and not even present anymore on recent batman-adv versions.

Reply:
What do you suggest, accessing it via `batctl originators`?

Reply:
Yes, use `batctl o -H` if you must. It still uses the debugfs because Gluon still runs on batman-adv 2018.1.

Reply:
Okay, will do.

Reply:
Changed in latest commit.